x11-wm/cwm/cwm-6.6.ebuild

```diff
@@ -1,35 +1,34 @@
-# Copyright 1999-2018 Gentoo Authors
+# Copyright 2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-inherit eutils toolchain-funcs
+inherit desktop eutils toolchain-funcs
 
 DESCRIPTION="OpenBSD fork of calmwm, a clean and lightweight window manager"
 HOMEPAGE="https://www.openbsd.org/cgi-bin/cvsweb/xenocara/app/cwm/
 	https://github.com/chneukirchen/cwm"
-SRC_URI="vanilla? ( http://chneukirchen.org/releases/${P}.tar.gz -> ${P}-chneukirchen.tar.gz )
-	!vanilla? ( https://github.com/xmw/cwm/tarball/ea9a436 -> ${P}-xmw.tar.gz )"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/chneukirchen/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/chneukirchen/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="amd64 arm64 x86"
+fi
 
 LICENSE="ISC"
 SLOT="0"
-KEYWORDS="amd64 arm x86"
-IUSE="vanilla"
 
 RDEPEND="x11-libs/libXft
 	x11-libs/libXinerama
-	x11-libs/libXrandr"
+	x11-libs/libXrandr
+"
 DEPEND="${RDEPEND}
 	virtual/pkgconfig
-	sys-devel/bison"
-
-src_unpack() {
-	default
-	# vcs-snapshot doesn't work with tarball names
-	if ! use vanilla ; then
-		mv *${PN}-* ${P} || die
-	fi
-}
+	sys-devel/bison
+"
```

Comment:
those dependencies are usually BDEPEND in EAPI=7

Reply:
Totally forgot, thanks for noticing, I've fixed that :)